Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby "2.0.0", patchlevel: "645"
+ruby "2.3.0"
```

Comment:
Can you remove this change? I'd rather discuss this separately in a different issue/PR.